assembly/assembly-ide-war/src/main/resources/org/eclipse/che/ide/IDE.gwt.xml

```diff
@@ -93,4 +93,5 @@
     <!-- Specify the paths for translatable code                    -->
     <source path='client'/>
     <source path='shared'/>
+    <collapse-all-properties />
```

Comment:
@vparfonov I just pushed an update of the patch in which I removed this extra-line.